script/build.js

```diff
@@ -5,3 +5,8 @@ const pattern = fs.readFileSync('./node_modules/emoji-test-regex-pattern/dist/la
 const output = input.replace('<% pattern %>', pattern) + '\n';
 
 fs.writeFileSync('./index.js', output);
+
+const inputModule = fs.readFileSync('./src/index.mjs', 'utf8').toString().trim();
+const outputModule = inputModule.replace("<% pattern %>", pattern) + "\n";
+
+fs.writeFileSync("./index.mjs", outputModule);
```

Comment:
To match the surrounding code style:

```suggestion
fs.writeFileSync('./index.mjs', outputModule);
```